.github/pull_request_template.md

```diff
@@ -1,12 +1,12 @@
-__IMPORTANT__
+# **IMPORTANT**
 
 In order to help other team mates in having context of the ticket and knowing which work has been made, it's mandatory to fill all the mandatory (*) sections (that includes replacing the text placeholders to describe properly the ticket). If any of them have not been filled in, the PR is not ready to review, so people will wait to do the review. (remove this note when you're sure all of the reqs. are fulfilled)
 
-## (*)Description
+## **Description**
```

Comment:
try to keep the (*) here and in the other places you removed it, as the description above explains that fields highlighted with the (*) are mandatory to be filled, so it's important we let people know the "must fill in"s